lighthouse-cli/bin.ts

```diff
@@ -114,6 +116,7 @@ Example: --output-path=./lighthouse-results.html`
   .default('output', Printer.GetValidOutputOptions()[Printer.OutputMode.pretty])
   .default('output-path', 'stdout')
   .default('port', 9222)
+  .default('timeout', 25)
```

Comment:
since the `MAX_WAIT_FOR_FULLY_LOADED` is still in driver, seems like we should let this value fall through as `undefined`? wdyt